### ISSUE TITLE
Support for condition coverage

### ISSUE THREAD

**Issue by nasava** [Bachmann1234/diff_cover#182]
Hallo, as far as I can see the condition/branch coverage reported by jacoco is not yet supported by diff_cover, isn't it?

How easy/hard would it be to add support? As far I can see up to now the tool is line oriented.
Maybe as a separate plugin which counts the lines plus the conditions. 
(Exactly this metric is used in my project for new code to be accepted into master)

**Comment by schinckel** [comment]
Yes, I just noticed that branch-coverage (as coverage.py calls it) is not reported upon.
